core/src/main/scala/kafka/coordinator/transaction/ProducerIdManager.scala

```diff
@@ -41,6 +43,7 @@ import scala.util.{Failure, Success, Try}
 object ProducerIdManager {
   // Once we reach this percentage of PIDs consumed from the current block, trigger a fetch of the next block
   val PidPrefetchThreshold = 0.90
+  val RetryBackoffMs = 100
```

Comment:
Would 50ms be enough? It seems like we are missing some configuration for backoff behavior for requests from brokers to the controller. It would be nice to have support exponential backoff as well.